tools/run_tests/dockerize/build_interop_image.sh

```diff
@@ -37,6 +37,7 @@ set -x
 #  INTEROP_IMAGE - name of tag of the final interop image
 #  BASE_NAME - base name used to locate the base Dockerfile and build script
 #  TTY_FLAG - optional -t flag to make docker allocate tty
+#  RUNTIME - runtime setting. If specified, Dockerfile-$RUNTIME will be used.
```

Comment:
I am not convinced this is useful. Right now our interop docker images have just one dimension - the name.  having grpc_interop_go17 and grpc_interop_go18 is equally powerful as having grpc_interop_go with two "runtime" flavors, but the latter is more complicated and requires more changes.
Also, introducing a concept of "runtime" is problematic because that can lead to confusion what counts as "runtime" vs what should already be a different language (and my past experience suggests people will be inconsistent in thinking one way or another).